rasa/core/training/story_reader/yaml_story_reader.py

```diff
@@ -161,17 +166,16 @@ def _parse_raw_user_utterance(self, step: Dict[Text, Any]) -> Optional[UserUtter
         raw_entities = step.get(KEY_STORY_ENTITIES, [])
         final_entities = YAMLStoryReader._parse_raw_entities(raw_entities)
 
-        if not user_utterance.startswith(INTENT_MESSAGE_PREFIX):
+        if user_utterance.startswith(INTENT_MESSAGE_PREFIX):
             common_utils.raise_warning(
                 f"Issue found in '{self.source_name}': \n"
-                f"User intent '{user_utterance}' should start with "
-                f"'{INTENT_MESSAGE_PREFIX}'. "
-                f"This story step will be skipped.",
+                f"User intent '{user_utterance}' starts with "
+                f"'{INTENT_MESSAGE_PREFIX}'. This is not required.",
                 docs=DOCS_URL_STORIES,
             )
-            return None
-        else:
+            # Remove leading slash
```

Comment:
Will there still be slash now when there is an `intent` key in the current yaml format?

Reply:
nope (this part of the diff is just because I cherry-picked them for now). I'll merge https://github.com/RasaHQ/rasa/pull/6117 and then the diff will go away